docs/source/configuration/authorization.mdx

```diff
@@ -98,199 +140,263 @@ So the `me` top level operation would not even be executed. When returning the r
 
-    products {
-        name
+```graphql title="Request scopes: 'user:read inventory:read'"
+query {
+  /* The following fields are filtered out
+  me { # Requires the "profile:read" scope
+    name
+    email
+  }
+  */
+  products {
+    name
+    amount
+    orders {
+      name
     }
+  }
 }
 ```
 
-And generate an error at path `/products/@/amount`.
+The response would include an error at the `/me` path.
 
-While if it presented the scopes `employee inventory`, it would filter the query as:
+## Composition and federation
 
-```graphql
-{
-    products {
-        name
-        amount
-        orders {
-            name
-        }
-    }
+Authorization directives are defined at the subgraph level and the router composes them in the supergraph schema. In other words, if subgraph fields or types include either `@authenticated` or `@requiresScopes` directives, they are set on the supergraph too.
+
+If multiple shared subgraph fields include `@requiresScopes`, the supergraph schema merges the sets of scopes required by each subgraph. For example, imagine one subgraph requires the `profile:read` scope on the `users` query:
+
+```graphql title="Subgraph A"
+type Query {
+  users: [User!]! @requiresScopes(scopes: ["profile:read"])
 }
 ```
 
-And generate an error at path `/me`.
+And another subgraph requires the `user:read` scope on `users` query:
 
+```graphql title="Subgraph B"
+type Query {
+  users: [User!]! @requiresScopes(scopes: ["user:read"])
+}
+```
 
-## Composition and federation
+Then the supergraph schema would require both scopes for it.
 
-The authorization directives are defined by the subgraph author, and will be carried into the supergraph schema:
-- `@authenticated`: if defined on a field or type by any of the subgraphs, it will be set in the supergraph too
-- `@requiresScopes`: same as `@authenticated`, and if there are multiple applications, the supergraph schema will merge the sets of scopes required by each subgraph
+```graphql title="Supergraph"
+type Query {
+  users: [User!]! @requiresScopes(scopes: ["profile:read", "user:read"])
+}
+```
 
-### Authorization and `@key`
+### Authorization and `@key` types
 
-When the authorization directives are set on fields used in `@key`, they will be usable by the router to join between subgraphs, but cannot be queried directly by the client. This behavior is similar to [Contracts](/graphos/delivery/contracts/) and the [`@inaccessible` directive](https://www.apollographql.com/docs/federation/federated-types/federated-directives/#inaccessible).
+The [`@key` directive](https://www.apollographql.com/docs/federation/entities/) lets you create an entity whose fields resolve across multiple subgraphs.
+If you use authorization directives on fields in [`@key` types](https://www.apollographql.com/docs/federation/entities/), Apollo still joins those fields between the subgraphs, but the client cannot query them directly.
```

Comment:
```suggestion
If you use authorization directives on fields in [`@key` directives](https://www.apollographql.com/docs/federation/entities/), Apollo still joins those fields between the subgraphs, but the client cannot query them directly.
```

The term `types` will be a bit overloaded here in the context of GraphQL since keys are just fields; probably clearer to retain `fields` instead.